packages/ticker/test/TickerPlugin.tests.ts

```diff
@@ -4,25 +4,25 @@ describe('TickerPlugin', () =>
 {
     interface App
     {
-        _ticker?: Ticker;
-        ticker?: Ticker;
+        _ticker?: Ticker | null;
+        ticker?: Ticker | null;
         render?(): void;
         start?(): void;
     }
     let app: App;
 
     it('should not start application before calling start method if options.autoStart is false', (done) =>
     {
-        const appp = {} as App;
+        const appp: App = {};
 
         TickerPlugin.init.call(appp, { autoStart: false });
 
         expect(appp.ticker).toBeInstanceOf(Ticker);
-        expect(appp.ticker.started).toBe(false);
+        expect(appp.ticker?.started).toBe(false);
```

Comment:
Sorry, maybe I don't get this part, but it feels this 2 lines have conflicting logic, no ?